.gitignore

```diff
@@ -1 +1,6 @@
-/resources
\ No newline at end of file
+.idea
+.vscode
+build
+cmake-build-debug
+.clang-format
```

Comment:
Btw I think its OK to leave it in index. Formatting is more or less repo property.